mandatory/1-fix-functions.js

```diff
@@ -32,9 +31,9 @@ function greaterThan10(num) {
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element= arr;
```

Comment:
we already have `arr `available so there is no need to save it to a new variable, doing `let element= arr[index]` here makes the element variable more useful :)